### PR TITLE
Update pointer example

### PR DESCRIPTION
**Pull request by alexbozhenko**
Pointers example were not working for me.
Update them.

### CODE REVIEW

File: _content/tour/pointers/example4.go [hunk 1]
```diff
@@ -54,21 +54,22 @@ func createUserV2() *user {
...
-./example4.go:24:6: cannot inline createUserV1: marked go:noinline
-./example4.go:38:6: cannot inline createUserV2: marked go:noinline
-./example4.go:14:6: cannot inline main: function too complex: cost 132 exceeds budget 80
-./example4.go:39:2: u escapes to heap:
-./example4.go:39:2:   flow: ~r0 = &u:
-./example4.go:39:2:     from &u (address-of) at ./example4.go:46:9
-./example4.go:39:2:     from return &u (return) at ./example4.go:46:2
-./example4.go:39:2: moved to heap: u
+# https://github.com/ardanlabs/gotour/blob/main/_content/tour/pointers/example4.go
+$ go build -gcflags -m=2 -tags OMIT example4.go
```
Comment (alexbozhenko) [inline _content/tour/pointers/example4.go:58]: without -tags, I was getting:
```package github.com/ardanlabs/gotour/_content/tour/pointers: build constraints exclude all Go files in```...

File: _content/tour/pointers/example4.go [hunk 2]
```diff
@@ -82,7 +83,7 @@ CALL	"".createUserV1(SB)
 
 // See bounds checking decisions.
 
-go build -gcflags="-d=ssa/check_bce/debug=1"
+$ go build -gcflags="-d=ssa/check_bce/debug=1"
```
Comment (alexbozhenko) [inline _content/tour/pointers/example4.go:86]: This produces empty output. Not sure how to fix...

File: _content/tour/pointers/example4.go [hunk 3]
```diff
@@ -82,7 +83,7 @@ CALL	"".createUserV1(SB)
 
 // See bounds checking decisions.
 
-go build -gcflags="-d=ssa/check_bce/debug=1"
+$ go build -gcflags="-d=ssa/check_bce/debug=1" example4.go
```
Comment (alexbozhenko) [inline _content/tour/pointers/example4.go:86]: This gives an empty output. Not sure how to fix...

**Comment by ardan-bkennedy** [comment]
All the commands worked for me when I was in the example4 folder and tried it.

```$ go build -gcflags -m=2
# github.com/ardanlabs/gotraining/topics/go/language/pointers/example4
./example4.go:24:6: cannot inline createUserV1: marked go:noinline
./example4.go:38:6: cannot inline createUserV2: marked go:noinline
./example4.go:14:6: cannot inline main: function too complex: cost 132 exceeds budget 80
./example4.go:39:2: u escapes to heap:
./example4.go:39:2:   flow: ~r0 = &u:
./example4.go:39:2:     from &u (address-of) at ./example4.go:46:9
./example4.go:39:2:     from return &u (return) at ./example4.go:46:2
./example4.go:39:2: moved to heap: u
```

**Comment by alexbozhenko** [comment]
@ardan-bkennedy but there is no pointers folder at
https://github.com/ardanlabs/gotraining/tree/master/topics/go

Did you run it in this repo? And it did not require `-tags OMIT` for you?

**Comment by ardan-bkennedy** [comment]
https://github.com/ardanlabs/gotraining/tree/master/topics/go/language/pointers

I did not need to add -tags OMIT.  That output I provided worked as copied.

**Comment by alexbozhenko** [comment]
Oh, ok. Sorry for the confusion.

**Comment by ardan-bkennedy** [comment]
THANK YOU for taking the time to report this issue. Please do it again if you find anything else.

**Pull request closed without merging**